schema.json

```diff
@@ -545,40 +586,7 @@
       "title": "End Device Profile",
       "description": "End device capabilities and boot parameters",
       "type": "object",
-      "examples": [
```

Comment:
Why remove?

Reply:
Thought it wasn't needed since vendorprofileid is gone but im stupid and didnt realize the other properties still apply 😇